pom.xml

```diff
@@ -82,6 +80,14 @@
             <groupId>org.osgi</groupId>
             <artifactId>osgi.core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.annotation.versioning</artifactId>
```

Comment:
we should always explicitly mention the scope "provided"

Reply:
done